### PR TITLE
DDF-2362 Move CatalogFrameworkImpl support classes to subpackage.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Moves the delegate and support classes in order to avoid clutter in the impl's package and better isolate the workers.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@ryeats @tbatie 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@pklinef

#### How should this be tested?
Full build/test/itest run is sufficient.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
DDF-2362

#### Screenshots (if appropriate)
N/A

#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/core/catalog-core-standardframework/pom.xml [hunk 1]
```diff
@@ -295,6 +295,7 @@
                             ddf.catalog.filter.impl,
                             ddf.catalog.history,
                             ddf.catalog.impl,
+                            ddf.catalog.impl.operations,
```
Comment (pklinef) [inline catalog/core/catalog-core-standardframework/pom.xml]: Did you actually need this?  I am not sure why this project is private packaging its own packages.

**Comment by pklinef** [comment]
👍

File: catalog/core/catalog-core-standardframework/pom.xml (hunk 1 above)
Comment (coyotesqrl) [inline catalog/core/catalog-core-standardframework/pom.xml]: I did. We may be able to clean up this bundle definition so it makes more sense, but as of right now this needed to be added as a private package or the bundle can't come up.

**Comment by emanns95** [comment]
Passed hero build :+1:

**Pull request merged**